### ISSUE TITLE
how to change the train parameter when i train a customdata content one class

### ISSUE THREAD

**Issue by zheng-2019** [zylo117/Yet-Another-EfficientDet-Pytorch#115]

**Comment by lambdayin** [comment]
11 classes on my datasets:

`import torch
pretrained_weights  = torch.load('./efficientdet-d0.pth')

pretrained_weights['classifier.header.pointwise_conv.conv.weight'].resize_(99,64,1,1)
pretrained_weights['classifier.header.pointwise_conv.conv.bias'].resize_(99,)
torch.save(pretrained_weights, './efficientdet-d0-sdxt.pth')`

"99" means  (num_classes * 11)

**Comment by fengz63** [comment]
My task is to detect foreign objects in images. All foreign objects belongs to the same class. That is to say there are only one class in my dataset. So what should I do to train the detection network.

**Comment by zylo117** [comment]
please refer to the tutorial

**Comment by fengz63** [comment]
> please refer to the tutorial

I have learned the tutorial. But my question is that how to train the classification network with just one class of data. And the loss of the classification network is no longer reduced when it falls to 0.32 on d0.
![image](https://user-images.githubusercontent.com/26649267/84113061-77180f00-aa5c-11ea-9d39-9660d09575d8.png)
Thank you for your reply!

**Comment by zylo117** [comment]
Is is a small dataset?
0.3 cls loss is quite high for one-class dataset.
Make sure the category id starts from 1 and try training the whole network instead of heads

**Comment by fengz63** [comment]
Yes, it is a small dataset. There are 8000 images in train set and 1000 images in validation set. 
I have transformed my annotations to coco style annotations. The category id is 1.
Ok, I will try to train the whole network instead of using only heads. Thank you!
